### PR TITLE
Release fixes

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -8,6 +8,7 @@
 !dfx_extension/**
 !examples/hello_world/**
 !examples/hello_world_candid_rpc/**
```
[the remark this answers is not in the document]
Comment (lastmjs) [reply .npmignore:10]: Can you double-check that there is no hello_world_candid_rpc anywhere in the examples/tests?

File: tests/end_to_end/candid_rpc/class_syntax/new/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "new_end_to_end_test_functional_syntax",
     "scripts": {
-        "test": "if [ \"$AZLE_TEST_RUN_ON_RELEASE\" == \"true\" ]; then tsx test/test.ts; else echo 'Skipping pretests'; fi"
+        "test": "if [ \"$AZLE_TEST_RUN_ON_RELEASE\" = \"true\" ]; then tsx test/test.ts; else echo 'Skipping pretests'; fi"
```
Comment (lastmjs) [inline tests/end_to_end/candid_rpc/class_syntax/new/package.json:4]: Doesn't this break it? If you run that locally doesn't it give an error? Didn't we run into this just a couple weeks ago?

Comment (bdemann) [reply tests/end_to_end/candid_rpc/class_syntax/new/package.json:4]: yes and we fixed it to a single equal on tests/end_to_end/http_server/new/package.json but we didn't update the class_syntax one

File: .npmignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (bdemann) [reply .npmignore]: Ah, now that I open up that folder I see that is is just left over, .azle, .dfx, node_modules, and target dirs and not an actual example. 

I have searched for it and removed it everywhere I found it

**Pull request merged**